tardis/gui.py

```diff
@@ -1,6 +1,39 @@
+"""Classes used to create the GUI for Tardis.
+
+This module contains all the classes and functions that create the GUI, but 
+it doesn't yet work outside IPython console. The console provides the event
+loop and the place to create/calculate the tardis model. So the module is
+basically a tool to visualize results. 
+
+Running instructions:
+--------------------
+    1. Decide which Qt binding you want to use (PySide or PyQt) and accordingly
+      set QT_API in shell::
+            export QT_API=pyside 
+            export QT_API=pyqt 
+    2. Start the IPython console with eventloop integration ::
+            ipython --pylab=qt4
+    3. Display your model::
+            from tardis import gui 
+            win = gui.Tardis()
+            win.show_model(mdl)
+
+Classes:
+-------
+    Tardis - The top level QMainWindow widget.
+
+Functions:
+---------
+    show_model(model) - A public method of Tardis(not gui itself) to take a 
+    model object and show its data.
+
+"""
+import os
+from pkg_resources import parse_version
+import exceptions
```

Comment:
Try to keep all imports at the top and imports convention should be something like #308. A small discussion regarding coding conventions we had.